ui/src/pages/version/components/modal/ModelVersionEndpointsTable.js

```diff
@@ -0,0 +1,132 @@
+/**
+ * Copyright 2020 The Merlin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { Fragment } from "react";
+import { Link } from "react-router-dom";
+import {
+  EuiInMemoryTable,
+  EuiText,
+  EuiHealth,
+  EuiIcon
+} from "@elastic/eui";
+import PropTypes from "prop-types";
+
+
+
+const defaultTextSize = "s";
+const defaultIconSize = "s"
+
+const ModelVersionEndpointsTable = ({ endpoints }) => {
+
+  const healthColor = status => {
+    switch (status) {
+      case "running":
+        return "orange";
+      case "pending":
+        return "gray";
+      case "terminating":
+        return "warning";
+      case "terminated":
+        return "warning";
+      case "failed":
+        return "danger";
+      case "failed_submission":
```

Comment:
We don't have all these states for the endpoint status, right? It's just these:

https://github.com/caraml-dev/merlin/blob/main/db-migrations/1_schema_creation.up.sql#L15